### ISSUE TITLE
[L4D2 Linux] Missing Fonts

### ISSUE THREAD

**Issue by Munoida** [ValveSoftware/Source-1-Games#4099]
The console spits out errors when the game first starts.
`Couldn't find custom font file 'resource/linux_fonts/DejaVuSans.ttf`
`Couldn't find custom font file 'resource/linux_fonts/DejaVuSans-Bold.ttf'`
`Couldn't find custom font file 'resource/linux_fonts/DejaVuSans-BoldOblique.ttf'`
`Couldn't find custom font file 'resource/linux_fonts/DejaVuSans-Oblique.ttf'`
`Couldn't find custom font file 'resource/linux_fonts/LiberationSans-Regular.ttf'`
`Couldn't find custom font file 'resource/linux_fonts/LiberationSans-Bold.ttf'`
`Couldn't find custom font file 'resource/linux_fonts/LiberationMono-Regular.ttf'`

Not sure what other games this affects but I know it affects Left 4 Dead. Not sure what it does, maybe some extra compatibility with non-latin names? All I know is that the fonts are missing from the resource folder and it does... absolutely nothing so far.

**Comment by Munoida** [comment]
Figured it out, it's just the previous fonts to be used for the captions/any other older VGUI Panel that doesn't have a specific vFont/Font for it

(Replaces the Captions with Bolded Italic version of Deja Vu Sans, I think the console font was also using Liberation Mono)
Not sure why the game still tries to check for those fonts but whatever lol, 

tldr: older vgui menus/elements will use the fonts specified in linux_fonts for whatever reason like with the source sdk base 2013 client, it's safe to ignore.
